internal/provider/provider.go

```diff
@@ -132,6 +132,7 @@ func Provider() *schema.Provider {
 			"packetfabric_marketplace_service_requests":  dataSourceVcRequests(),
 			"packetfabric_virtual_circuits":              datasourceBackboneServices(),
 			"packetfabric_point_to_point":                datasourcePointToPoint(),
+			"packetfabric_locations_region":              datasourceLocationsRegions(),
```

Comment:
Should it be named `packetfabric_locations_region` or `packetfabric_locations_regions`?

Reply:
We have been using singular so far for other resources so let's go with `packetfabric_locations_region` to be consistent.

Reply:
`packetfabric_locations_regions` is also more consistent with the API